src/frontend/qt_sdl/main.cpp

```diff
@@ -1399,16 +1403,52 @@ QString MainWindow::loadErrorStr(int error)
     }
 }
 
+std::string extractROM(char* zipName, std::string zipDir){
+    //Open the ZIP archive
+    int err = 0;
+    zip *z = zip_open(zipName, 0, &err);
+    
+	struct zip_stat st;
```

Comment:
inconsistent identation

Reply:
Weird. The indentation is fine on Notepad++

Reply:
fixed.